crates/debug/src/transform/expression.rs

```diff
@@ -600,13 +600,19 @@ impl<'a, 'b> ValueLabelRangesBuilder<'a, 'b> {
 mod tests {
     use super::compile_expression;
     use super::{AddressTransform, FunctionFrameInfo, ValueLabel, ValueLabelsRanges};
-    use gimli::{self, Encoding, EndianSlice, Expression, RunTimeEndian};
+    use gimli::{self, constants, Encoding, EndianSlice, Expression, RunTimeEndian};
     use wasmtime_environ::CompiledFunction;
 
+    macro_rules! dw_op {
+        (DW_OP_WASM_location) => { 0xed };
```

Comment:
Allow `DW_OP_WASM_location 0x0 20` and other multi-arg opcodes?

Reply:
Maybe. But I would limit first arg too (e.g. similar to https://yurydelendik.github.io/webassembly-dwarf/#DWARF-expressions-and-location-descriptions): `DW_OP_WASM_location local ..` or `DW_OP_WASM_location 0x1 ..`

Reply:
I guess we should get fancy in a follow-up. This is testing code, anyway. (But it was useful to learn a macro technique.)